snap/snapcraft.yaml

```diff
@@ -637,3 +640,13 @@ parts:
         sudo rm /usr/local/bin/go /usr/local/bin/gofmt
         sudo ln -sf /usr/local/go1.14/bin/go /usr/local/bin/go
         sudo ln -sf /usr/local/go1.14/bin/gofmt /usr/local/bin/gofmt
+    socket-proxy:
+      plugin: cmake
+      source: https://github.com/PelionIoT/socket-proxy.git
+      source-type: git
+      source-tag: v1.0.2
+      override-build: |
+        snapcraftctl build
+        install ${SNAPCRAFT_PROJECT_DIR}/files/socket-proxy/scripts/launch-socket-proxy.sh ${SNAPCRAFT_PART_INSTALL}/
+        install -d ${SNAPCRAFT_PART_INSTALL}/wigwag/etc
+        git describe --tags > ${SNAPCRAFT_PART_INSTALL}/wigwag/etc/socket-proxy.VERSION
```

Comment:
Looks like the print-versions.sh script needs to be modified too: https://github.com/PelionIoT/snap-pelion-edge/blob/dev/files/version/print-versions.sh

Reply:
done.